server/gc/src/test/java/org/apache/accumulo/gc/SimpleGarbageCollectorTest.java

```diff
@@ -94,6 +94,7 @@ public void testInit() {
     assertTrue(gc.isUsingTrash());
     assertEquals(1000L, gc.getStartDelay());
     assertEquals(2, gc.getNumDeleteThreads());
+    assertFalse(gc.inSafeMode()); //false by default
```

Comment:
When building the PR, the formatter added a space between `//` and `false by default`. Probably good to run the formatter before merging to avoid a second formatting-only commit to fix.